GameData/KerbalismConfig/Profiles/ROKerbalism.cfg

```diff
@@ -654,7 +654,7 @@ Supply
 			name = Vacuum Scrubber
 			desc = A dual-bed vacuum-exposing regenerative scrubber that sequesters <b>CarbonDioxide</b> from the internal atmosphere.
 			tech = basicLifeSupport
-			mass = 0.03 //FIXME
```

Comment:
Matching mass with advanced vacuum scrubber, it still requires more power to run.